openerp/addons/base/res/res_company.py

```diff
@@ -229,13 +229,14 @@ def _company_default_get(self, cr, uid, object=False, field=False, context=None)
         if not context:
             context = {}
         proxy = self.pool.get('multi_company.default')
+        user = self.pool.get('res.users').browse(cr, SUPERUSER_ID, uid, context=context)
```

Comment:
I saw that it is used at multiple places, but I don't really like it since the function starts with an underscore. Would be a slight performance improvement in terms of cache prefetching though, and avoid using SUPERUSER_ID. To avoid calling that undercore-prefixed function, the best solution would be a read but you would have to check the returned dict... which is exactly what this _get_company method does already so... :)